apps/mobile/mobile-app/components/Themed.tsx

```
@@ -3,43 +3,46 @@
-    return colorFromProps;
-  } else {
-    return Colors[theme][colorName];
-  }
+	const theme = useColorScheme() ?? 'light'
+	const colorFromProps = props[theme]
+
+	if (colorFromProps) {
+		return colorFromProps
+	} else {
+		return Colors[theme][colorName]
+	}
```

Biome (1.9.4)

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)